diff --git a/config.def.h b/config.def.h
index 91ab8ca..ccf91ef 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Liberation Mono:pixelsize=25::antialias=true:autohint=false:dpi=254";
 static int borderpx = 2;

 /*
