diff --git a/config.def.h b/config.def.h
index 91ab8ca..6d507c3 100644
--- a/config.def.h
+++ b/config.def.h
@@ -53,8 +53,8 @@ int allowwindowops = 0;
  * near minlatency, but it waits longer for slow updates to avoid partial draw.
  * low minlatency will tear/flicker more, as it can "detect" idle too early.
  */
-static double minlatency = 8;
-static double maxlatency = 33;
+static double minlatency = 1;
+static double maxlatency = 1;

 /*
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
